nodes/nomos-node/Cargo.toml

```diff
@@ -51,5 +51,7 @@ tower-http = { version = "0.4", features = ["cors", "trace"] }
 time = "0.3"
 
 [features]
+default = ["tracing"]
```

Comment:
If we set it by default, how would we deactivate it? Is there a no-features flag when compiling? 🤔

Reply:
> To build a node without tracing use cargo build -p nomos-node --no-default-features.

Reply:
> > To build a node without tracing use cargo build -p nomos-node --no-default-features.

Ah 💩 , went to the code directly. Perfec!